src/fragmentarium/ui/search/FragmentariumSearchResults.test.tsx

```diff
@@ -104,14 +111,52 @@ describe('search fragmentarium only transliteration', () => {
       )
     }
   })
-
   it('Displays script', () => {
     for (const fragment of fragments) {
       expect(screen.getAllByText(`(${fragment.script})`)).not.toEqual([])
     }
   })
-
   it('Displays matching lines', () => {
     expect(screen.getByText('kur')).toBeVisible()
   })
 })
+
+let CUT_OFF: number
```

Comment:
`CUT_OFF` is basically the number of items per page, right?

Reply:
yes